unitTests/ui/ResultListTest.ts

```diff
@@ -784,11 +784,48 @@ export function ResultListTest() {
         describe('enableScrollToTop set to', () => {
           let infiniteScrollContainer: HTMLElement;
           let allResult: IQueryResults;
+          let scrollToSpy: jasmine.Spy;
+          let scrollTopSpy: jasmine.Spy;
+
           beforeEach(() => {
-            infiniteScrollContainer = { scrollTop: 500 } as HTMLElement;
+            document.createElement('div');
+            infiniteScrollContainer = document.createElement('div');
+            infiniteScrollContainer.style.height = '10000px';
+            infiniteScrollContainer.style.overflow = 'scroll';
+            document.body.appendChild(infiniteScrollContainer);
+            scrollToSpy = spyOn(infiniteScrollContainer, 'scrollTo');
+            let pValue = 500;
```

Comment:
`cons` please. 

Also what is `pValue` ? 
Do you mean `positionValue` ?

I'd suggest a more descriptive name, like `scrollTopPosition`

Reply:
I'll take that suggestion.
It's a let because you need to be able to change the value with the closure created by **Object.defineProperty**